src/main/kotlin/de/itemis/mps/gradle/downloadJBR/Plugin.kt

```diff
@@ -0,0 +1,67 @@
+package de.itemis.mps.gradle.downloadJBR
+
+import org.gradle.api.GradleException
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.tasks.Copy
+import org.apache.tools.ant.taskdefs.condition.Os
+import java.io.File
+
+open class DownloadJBRConfiguration {
```

Comment:
Casing is inconsistent: JBR in `DownloadJBRConfiguration` vs Jbr in `DownloadJbrProjectPlugin`.